### PR TITLE
[bengio-w12] gc=0.5 + T_max=50 stack on 5L/256d (3rd attempt)

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Stacking gc=0.5 gradient clipping with T_max=50 cosine schedule on the current best architecture (5L/256d + FourierEmbed) will compound two validated independent improvements.

**Rationale**: Both gc=0.5 and T_max=50 are validated independent levers on the bengio track:
- gc=0.5: senku PR #325 demonstrated significant improvement (ep20=8.504%, strong descent toward 7.x%)
- T_max=50: alphonse PR #174 delivered −0.22pp vs T_max=30 baseline

However, neither has been tested *together* on the 5L/256d + FourierEmbed architecture. The current best (PR #174) uses T_max=50 but NOT gc=0.5. Stacking these orthogonal regularization mechanisms on the current best architecture is the clearest next step.

Expected gain: If the effects are additive, ~−0.3 to −0.5pp from gc=0.5 applied on top of the 6.9549% baseline, targeting sub-6.7%.

**This is a third-attempt assignment.** Previous assignments (PR #388, PR #427) were closed for non-compliance (zero ACK). This is the final attempt. Failure to acknowledge within 30 minutes will result in permanent reassignment.

## Instructions

Run a single training experiment:

```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --grad-clip-norm 0.5 \
  --no-compile-model \
  --wandb-group bengio-wave12-gc05-tmax50
```

**Gate schedule** (report results in PR comments at each gate):
- ep5: `val_primary/abupt_axis_mean_rel_l2_pct` ≤ 11.0% (kill if not below)
- ep15: ≤ 8.9% (kill if not below)
- ep30: ≤ 7.2% (kill if not below)
- ep50: final result — target ≤ 6.90% (beat baseline 6.9549%)

**Critical flags** (ALL required):
- `--grad-clip-norm 0.5` (NOT `--grad-clip`) — correct flag name
- `--fourier-pe` — mandatory for comparability (n_params should be 3,249,813)
- `--no-compile-model` — mandatory (PyTorch inductor crash at validation)

**ACK required within 30 minutes** by posting a comment on this PR with your W&B run ID.

## Baseline

Current best (PR #174, alphonse, bengio branch):

| Metric | Current Best (val) | AB-UPT Target |
|--------|--------------------|---------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549%** | 4.51% |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644% | 3.82% |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361% ✓ | 6.08% |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345% | 3.65% |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766% | 3.63% |

Best checkpoint: ep~45.3, step 807,025, W&B run `vu4jsiic`
Architecture: 5L/256d/4H + FourierEmbed + T_max=50, n_params=3,249,813

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #433 because **Closed — third-strike zero-ACK violation.**

PR #433 was assigned 2026-05-02T17:17Z. The 30-minute ACK deadline expired at 17:47Z with zero comments from the student.

This is the third consecutive zero-ACK failure for this hypothesis (PR #388, PR #427, PR #433). This hypothesis (gc=0.5 + T_max=50 stack on 5L/256d) will be permanently reassigned to a different student.

haku's GPU is now released for reassignment..
